serveradmin/serverdb/query_committer.py

```diff
@@ -108,6 +119,8 @@ def commit_query(created=[], changed=[], deleted=[], app=None, user=None):
         _upsert_attributes(attribute_lookup, changed, changed_servers)
         changed_objects = _materialize(changed_servers, joined_attributes)
 
+        # @TODO Improve this function by checking only attributes of ACLs that
```

Comment:
Here there is suddenly a `@` in front of the "TODO". Is that on purpose?

Reply:
No it's just what my IDE does automatically. I removed it.